Makefile

```diff
@@ -33,7 +33,7 @@ cli: bin/$(GOOS)-$(GOARCH)/viam-cli
 cli-ci: bin/$(GOOS)-$(GOARCH)/viam-cli
 	if [ -n "$(CI_RELEASE)" ]; then \
 		mkdir -p bin/deploy-ci/; \
-		cp $< bin/deploy-ci/viam-cli-$(CI_RELEASE)-$(GOOS)-$(GOARCH); \
+		cp $< bin/deploy-ci/viam-cli-$(CI_RELEASE)-$(GOOS)-$(GOARCH)$(EXE_SUFFIX); \
```

Comment:
[opt] and kind of a flyby change, but this make target should probably be the file we create (e.g. `bin/deploy-ci/viam-cli-$(CI_RELEASE)-$(GOOS)-$(GOARCH)$(EXE_SUFFIX):`) instead being phony.

your call if you wanna change this now

Reply:
this was originally the name of the target, got changed to phony in a previous review

Reply:
gotcha, ignore my comment then!